src/main/java/com/cryptlex/lexfloatclient/LexFloatClient.java

```diff
@@ -113,6 +113,88 @@ public static void SetFloatingClientMetadata(String key, String value) throws Le
         }
     }
 
+
+    /**
+     * Gets the product version name.
+     * 
+     * @return name - Returns the name of the Product Version being used.
+     * @throws LexFloatClientException
+     * @throws UnsupportedEncodingException
+     */
+     public static String GetProductVersionName() throws LexFloatClientException, UnsupportedEncodingException {
+        int status;
+        if (Platform.isWindows()) {
+            CharBuffer buffer = CharBuffer.allocate(256);
+            status = LexFloatClientNative.GetProductVersionName(buffer, 256);
+            if (LF_OK == status) {
+                return buffer.toString().trim();
+            }
+        } else {
+            ByteBuffer buffer = ByteBuffer.allocate(256);
+            status = LexFloatClientNative.GetProductVersionName(buffer, 256);
+            if (LF_OK == status) {
+                return new String(buffer.array(), "UTF-8").trim();
+            }
+        }
+        throw new LexFloatClientException(status);
+    }
+
+    /**
+     * Gets the product version display name.
+     * 
+     * @return displayName - Returns the display name of the Product Version being
+     *         used.
```

Comment:
please remove the additional space

Reply:
We have followed this format (space between the description line and prams, returns....) throughout the file.